app/models/amt/AMTConditionTable.scala

```diff
@@ -28,6 +33,29 @@ class AMTConditionTable(tag: Tag) extends Table[AMTCondition](tag, Some("sidewal
 object AMTConditionTable {
   val db = play.api.db.slick.DB
   val amtConditions = TableQuery[AMTConditionTable]
+  val amtAssignments = TableQuery[AMTAssignmentTable]
+  val max_num_condition_assignments = 5
```

Comment:
Put this in camelCase and specify data type
val maxNumConditionAssignments: Int = 5